### PR TITLE
Update Slack Docs

### PR DESCRIPTION
**Pull request by emilyrohrbough**

### CODE REVIEW

File: docs/guides/cloud/integrations/slack.mdx [hunk 1]
```diff
@@ -1,228 +1,143 @@
...
 
-   <DocsImage
-     src="/img/guides/cloud/organizations/navigate-to-organization-integrations.jpg"
-     alt="Install Cypress Slack from Integrations"
-     width={500}
-   />
+The Slack messages can be tailored to match your teams' needs.
 
-4. Click the **Enable** button in the Slack section.
+- The Slack app can send results to one or more public or private Slack channels or directly to a specific user
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:44]: Is 'Slack app' right here?

File: docs/guides/cloud/integrations/slack.mdx [hunk 2]
```diff
@@ -1,228 +1,143 @@
...
-   all projects in your organization to the specified Slack channel.
+The following describes where to go to configure Slack for an organization and a project. The remaining sections will describe the supported configuration options.
 
-For more information about how the Cypress integration for Slack collects and
-manages your information, please visit our
-[Privacy Policy](https://on.cypress.io/privacy-policy).
+#### Organization settings
 
-## Per-organization configuration
+To share Slack configuration for an organization:
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:54]: Is 'share' the right word here? Did you mean 'update' or 'configure'?

File: docs/guides/cloud/integrations/slack.mdx [hunk 3]
```diff
@@ -1,228 +1,143 @@
...
+#### Project settings
 
-1. Navigate to the **Integrations** page for the organization with the installed
-   integration.
-2. Within the Slack integration, click **Configure**.
-3. Click **Add Slack Channel**.
-4. You'll see a popup window that allows you to choose the channel to associate
-   with the organization. Cypress Cloud will post run results for all projects
-   in your organization to the new Slack channel.
+To share Slack configuration for a project:
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:62]: 'share'?

File: docs/guides/cloud/integrations/slack.mdx [hunk 4]
```diff
@@ -1,228 +1,143 @@
...
+4. After you approve the app permissions, you can choose your Slack workspace and channel to
+   associate with the installation.
+5. The installation is complete!
 
-   <DocsImage
-     src="/img/guides/cloud/organizations/select-cypress-organization.jpg"
-     alt="Select an organization"
-     width={500}
-   />
+Now Cypress will post run results for all projects in your organization to the specified Slack channel.
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:38]: Can you do the setup without it turning on all results for the entire org to start? Like can you enable it per project to start? Or is this step required to turn it on for an org entirely.

File: docs/guides/cloud/integrations/slack.mdx [hunk 5]
```diff
@@ -1,228 +1,143 @@
...
-3. Under **Notifications**, select your preference for each Slack channel:
+You can have Cypress Cloud post run results to specific channels. To add a
+channel:
 
-- **Failed runs**: will notify on runs with a failed status.
-- **Passed runs**: will notify on runs with a passed status.
-- **Flaky tests**: will notify on runs that have an identified flaky test.
+1. Navigate to the Slack configuration page
+2. Click **Add Slack Channel**.
+3. You'll see a popup window that allows you to choose the channel or individual associate
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:75]: 'individual associate', not sure what you mean here. You mean you can post to private messages?

File: docs/guides/cloud/integrations/slack.mdx [hunk 6]
```diff
@@ -1,228 +1,143 @@
...
-and resume notifications for a specific channel without losing the configuration
-you've put in place.
+### Direct messages
 
-1. Navigate to the **Integrations** page for the organization with the installed
-   integration.
-2. Within the Cypress integration for Slack, click **Configure**.
-3. Under **Actions**, select your **Mute** for each Slack channel you want
-   muted.
+You can have Cypress Cloud post run results to specific user. To post to a user:
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:83]:
```suggestion
You can have Cypress Cloud post run results to a specific direct message To post to a direct message:
```

File: docs/guides/cloud/integrations/slack.mdx [hunk 7]
```diff
@@ -1,228 +1,143 @@
...
 
-### Remove a Slack channel
+1. Navigate to the Slack configuration page
+2. Click **Add Slack Channel**.
+3. You'll see a popup window that allows you to choose the channel or individual associate
+   with the organization.
+4. Search for and selecting the user that messages should be sent to.
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:89]:
```suggestion
4. Search for and select the user that messages should be sent to.
```

File: docs/guides/cloud/integrations/slack.mdx [hunk 8]
```diff
@@ -1,228 +1,143 @@
...
 
-You can have Cypress Cloud stop posting notifications to a channel. You can
-remove all Slack channels if you'd prefer to disable global notifications
-altogether in favor of per-project notifications.
+Cypress Cloud will begin to send run results to the this user.
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:91]:
```suggestion
Cypress Cloud will begin to send run results to this user.
```

File: docs/guides/cloud/integrations/slack.mdx [hunk 9]
```diff
@@ -1,228 +1,143 @@
...
-you've put in place. You can even mute the messages for the global organization
-channels!
+you've put in place.
 
-1. Navigate to the **Integrations** page for the project with the installed
-   integration.
-2. Scroll down to the **Slack Integration** section.
-3. Under **Actions**, select your **Mute** for each Slack channel you want
+1. Navigate to the Slack configuration page
+2. Under **Actions**, select your **Mute** for each Slack channel you want
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:124]:
```suggestion
2. Under **Actions**, select **Mute** for each Slack channel you want
```

File: docs/guides/cloud/integrations/slack.mdx [hunk 10]
```diff
@@ -1,228 +1,143 @@
...
+You can have Cypress Cloud stop posting notifications to a channel. You can
+remove all Slack channels if you'd prefer to disable global notifications
+altogether in favor of per-project notifications.
 
-1. Navigate to the **Integrations** page for the project with the installed
-   integration.
-2. Scroll down to the **Slack Integration** section.
-3. Under **Actions**, select your **Delete** for each Slack channel you want
+1. Navigate to the Slack configuration page
+2. Under **Actions**, select your **Delete** for each Slack channel you want
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:134]:
```suggestion
2. Under **Actions**, select **Delete** for each Slack channel you want
```

File: docs/guides/cloud/integrations/slack.mdx [hunk 11]
```diff
@@ -231,3 +146,22 @@ will be able to install the Cypress integration for Slack again in the future.
...
+## Security and compliance
+
+Review our [Privacy Policy](https://on.cypress.io/privacy-policy) for information about how the Cypress app for Slack collects and
+manages your information.
+
+### Authorizations
+
+When authorized, Cypress can:
+
+- View information about a user’s identity
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:159]: Can we be more specific here? This sounds really broad. Like, you can see the user's first name/last name? Or their entire profile?

File: docs/guides/cloud/integrations/slack.mdx [hunk 12]
```diff
@@ -231,3 +146,22 @@ will be able to install the Cypress integration for Slack again in the future.
...
+- Post messages to specific channels in Slack & direct message conversations
+- Show previews of cloud.cypress.io URLs in messages
+- View URLs from cloud.cypress.io
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:162]: What? I don't understand this. We are authorized to see what data from the user's slack here? We already have urls from our own cloud account...

File: docs/guides/cloud/integrations/slack.mdx [hunk 13]
```diff
@@ -231,3 +146,22 @@ will be able to install the Cypress integration for Slack again in the future.
...
+
+### App scopes
```
Comment (jennifer-shehane) [inline docs/guides/cloud/integrations/slack.mdx:164]: Can you link to some doc that explains the definitions of scopes?

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]

File: docs/guides/cloud/integrations/slack.mdx (hunk 1 above)
Comment (emilyrohrbough) [reply docs/guides/cloud/integrations/slack.mdx:44]: Slack app came from Slack's documentation: https://api.slack.com/docs/apps

File: docs/guides/cloud/integrations/slack.mdx (hunk 11 above)
Comment (emilyrohrbough) [reply docs/guides/cloud/integrations/slack.mdx:159]: This is what Slack shares when reviewing the app
Screenshot 2024-09-25 at 10 12 56 AM

File: docs/guides/cloud/integrations/slack.mdx (hunk 2 above)
Comment (emilyrohrbough) [inline docs/guides/cloud/integrations/slack.mdx:54]:
```suggestion
Configure Slack for the organization:
```

File: docs/guides/cloud/integrations/slack.mdx (hunk 3 above)
Comment (emilyrohrbough) [inline docs/guides/cloud/integrations/slack.mdx:62]:
```suggestion
Configurate Slack for a project:
```

File: docs/guides/cloud/integrations/slack.mdx (hunk 6 above)
Comment (emilyrohrbough) [inline docs/guides/cloud/integrations/slack.mdx:83]:
```suggestion
You can have Cypress Cloud post run results as a direct message to a single user. To post to a direct message:
```

File: docs/guides/cloud/integrations/slack.mdx [hunk 14]
```diff
@@ -231,3 +146,22 @@ will be able to install the Cypress integration for Slack again in the future.
...
+
+- User token scopes: `links:read`, `links:write`
+- Bot token scopes: `chat:write`, `incoming-webhook`, `links:read`, `links:write`
```
Comment (emilyrohrbough) [inline docs/guides/cloud/integrations/slack.mdx:167]:
```suggestion
- Bot token scopes: [`chat:write`](https://api.slack.com/scopes/chat:write), [`incoming-webhook`](https://api.slack.com/scopes/incoming-webhook), [`links:read`](https://api.slack.com/scopes/links:read), [`links:write`](https://api.slack.com/scopes/links:write)
```

**Comment by MikeMcC399** [comment]
The https://www.cypress.io/privacy-policy referred to in https://docs.cypress.io/guides/cloud/integrations/slack has some formatting issues. Perhaps you could pass this on?

image

**Comment by jennifer-shehane** [comment]
Closing in favor of https://github.com/cypress-io/cypress-documentation/pull/5961/files#diff-362dac2dac5a6282a1c4fdf42e0b0427ca92941d38d0db46b27b84289c2fbd86L1

**Pull request closed without merging**